### PR TITLE
Create a collection

### LINKED ISSUE #143: Create new collection
From the dashboard page, I should be able to click the "+ Create a new collection" button. See [mockup](https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/426926001). When I click the button, I am taken to the [new collection form](https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/420930437). For this issue, we do not expect the implementation to build out the entire form as displayed in the mock-up; it suffices to implement a minimal version of the form focusing on the required elements:

* Name
* Description
* Contact Email
* Visibility
* Managers

When I complete creating the collection, it is displayed as expected in the dashboard.

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
Until a user creates a collection, there is nothing to deposit into.  
Fixes #143

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Ruby dependencies can be installed with `bundle install`, JavaScript dependencie
 
 Start up dependencies with `docker-compose up`, then run tests with `bundle exec rspec`.
 
-To run the server in development mode set `REMOTE_USER` because we aren't running behing Shibboleth.
+To run the server in development mode set `REMOTE_USER` because we aren't running behind Shibboleth.
```
Comment (mjgiarlo) [inline README.md:22]: Oops. Good catch.

File: app/controllers/collections_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  def create
+    @collection = Collection.new(collection_params.merge(creator: current_user))
+    authorize! @collection
+
+    if @collection.save
+      # TODO: https://github.com/sul-dlss/happy-heron/issues/92
+      # DepositCollectionJob.perform_later(@collection) if params[:commit] == 'Deposit'
+      redirect_to dashboard_path
+    else
+      render :new
```
Comment (mjgiarlo) [inline app/controllers/collections_controller.rb:24]: Worth tossing a request spec at this case?

**Review by mjgiarlo — COMMENTED** [review]
Looks good. Have a question about a small unhappy path test.

File: app/controllers/collections_controller.rb (hunk 2 above)
Comment (jcoyne) [reply app/controllers/collections_controller.rb:24]: Not possible yet because we don't have any validations yet.

Comment (mjgiarlo) [reply app/controllers/collections_controller.rb:24]: No biggie.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**